packages/admin/admin/src/translator/config/ContentTranslationServiceContext.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export interface TranslationConfig {
+    enabled: boolean;
+    translate: (input: string) => Promise<string | void>;
```

Comment:
why void?

Reply:
True it should just be string, added void because of the "No operation"  function defaultValue. Changed it to return the input value by default. Although this default translate function will never be used, only the enabled param if the project does not wrap an use-case with the Provider gets used. Else the translate function gets overwritten since its mandatory to set in the Provider.

```
export const ContentTranslationServiceContext = React.createContext<TranslationConfig>({
    enabled: false,
    translate: async (input) => input,
});
```

instead of 

```
translate: async () => {
        /* Noop */
    },
```

Reply:
alternatively you could throw an error in the default translate function

Reply:
I'd prefer throwing an error